crates/cargo-contract/src/cmd/extrinsics/events.rs

```diff
@@ -107,9 +107,18 @@ impl DisplayEvents {
                 ) && field.name() == Some("data")
                 {
                     tracing::debug!("event data: {:?}", hex::encode(&event_data));
-                    let contract_event = transcoder.decode_contract_event(event_data)?;
-                    let field = Field::new(String::from("data"), contract_event);
-                    event_entry.fields.push(field);
+                    match transcoder.decode_contract_event(event_data) {
+                        Result::Ok(contract_event) => {
```

Comment:
```suggestion
                        Ok(contract_event) => {
```

Reply:
unfortunately, we already import method `Ok()` from `anyhow` and there would be a collision

Reply:
That looks like a mistake, please remove that import.

Reply:
Ok :)